update/Pos/Update/Worker.hs

```diff
@@ -67,14 +69,22 @@ checkForUpdate = do
                 svNumber . upSoftwareVersion . cpsUpdateProposal
         let newestCPS =
                 maximumBy (comparing cpsToNumericVersion) confirmedProposals
-        logInfo $
-            sformat
-                ("There are new confirmed update proposals for our application: "
-                 %listJsonIndent 2%
-                 "\n The newest one is: "%build%" and we want to download it")
-                (cpsUpdateProposal <$> confirmedProposals)
-                (cpsUpdateProposal newestCPS)
-        downloadUpdate newestCPS
+
+        ifM (noServersProvided newestCPS) (logInfo "There is no servers provided") $ do
+            logInfo $
+                sformat
+                    ("There are new confirmed update proposals for our application: "
+                    %listJsonIndent 2%
+                    "\n The newest one is: "%build%" and we want to download it")
+                    (cpsUpdateProposal <$> confirmedProposals)
+                    (cpsUpdateProposal newestCPS)
+            downloadUpdate newestCPS
+
+    noServersProvided :: ConfirmedProposalState -> m Bool
+    noServersProvided ConfirmedProposalState {..} = do
```

Comment:
Maybe I don't understand something, but it seems to me that `ConfirmedProposalState` is not used in this function.

Reply:
And btw, I don't understand why this change is part of this PR, I expected it to be part of CSL-1602 (and it is, but for some reason it's in both PRs).